genx/make_deb.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh
+
+# exit on error
+set -e
+
+python setup.py bdist_rpm
+
+# These commands requires dpkg-dev alien
+echo "Moving distribution files..."
+
+cd dist
+
+NAME=$(basename *.tar.gz .tar.gz)
```

Comment:
This will fail when ever there is more than one .tar.gz file in the dist folder.